cmd/odf/restore/crds.go

```diff
@@ -20,8 +62,19 @@ var deletedCmd = &cobra.Command{
 	},
 	Run: func(cmd *cobra.Command, args []string) {
 		k8sutil.SetDeploymentScale(cmd.Context(), root.ClientSets.Kube, root.OperatorNamespace, "ocs-operator", 0)
-		pkgrestore.RestoreCrd(cmd.Context(), root.ClientSets, root.OperatorNamespace, root.StorageClusterNamespace, args)
+		// Parse the fully qualified CRD (e.g. "cephclusters.ceph.rook.io").
+		resourceName, groupName, version, err := parseFullyQualifiedCRD(args[0])
+		if err != nil {
+			fmt.Printf("Error parsing CRD type: %v\n", err)
```

Comment:
Instead of printing the error, see the helper for `logging.Fatal()`

Reply:
done